PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -519,9 +519,13 @@ export class ParaReportPublishService {
             return;
         }
         await this.processXUnitResults();
-        await this.processSarifResults();
-        await this.processCoberturaResults();
-       
+
+        if (this.pipelineType == PipelineTypeEnum.RELEASE) {
```

Comment:
I think we can do the condition check and skipping earlier, before transformation of sarif and coverage reports. 
@rwang-parasoft